controller/common/node_info.py

```diff
@@ -53,7 +61,7 @@ def is_array_iscsi_iqns_match(self, host_iqns):
            Is current host iqns matches
         """
         host_iqns_lower = [iqn.lower() for iqn in host_iqns]
-        return self._iscsi_iqn_lowercase in host_iqns_lower
+        return not self._iscsi_iqns_lowercase.isdisjoint(host_iqns_lower)
```

Comment:
could this be moved to a common private wrapper `self._is_match(self._iscsi_iqns_lowercase, host_iqns)` method?

Reply:
done

Reply:
I meant that `_is_match` would be private, so methods like `is_array_wwns_match` would call it

Reply:
why do we need those methods?

Reply:
so we can keep the members private, and to reduce arguments (both simplify the API IMO).
I think it would also reduce the diff

Reply:
why to keep the members private? I think its a good thing that they are public
and I think its pretty simple this way

Reply:
because `develop` already has several lines like this one:
```
if initiators.is_array_wwns_match(host_wwns):
```
and after your change, they will look like this one:
```
if initiators.is_match(initiators.fc_wwns, host_wwns):
```
which creates an unnecessary diff (the diff is already large enough) and a more cumbersome call (adding another `initiators.` per call).

if you still think it's better, I think we can discuss it in a separate ticket

Reply:
ok, done